aries-site/src/layouts/main/Header.js

```diff
@@ -23,11 +22,27 @@ const StyledHeader = ({ ...rest }) => {
       {...rest}
     >
       <Link href="/" passHref>
-        <AppIdentity
```

Comment:
Should we retain an `<AppIdentity />` layout component, even if it is just used for the site? Easy to spot and know what it stands for. Also a pattern we might want to reference when implementing elsewhere and easier to search for AppIdentity.

Reply:
Yes, that would work.

Reply:
I essentially took what lives in the aries-core file and brought it locally so we're not importing anything from aries-core.